examples/flutter_view/pubspec.yaml

```diff
@@ -5,55 +5,17 @@ dependencies:
   flutter:
     sdk: flutter
 
-  args: 1.4.1 # TRANSITIVE DEPENDENCY
```

Comment:
I wonder: how were these transitive deps here to begin with when there are no direct dev dependencies here?